terraform/aws/variables.tf

```diff
@@ -24,10 +38,18 @@ variable "servers" {
 }
 
 variable "instance_type" {
-    default = "m1.small"
-    description = "The instance type to launch."
+    default = {
+        ubuntu = "m1.small"
+        rhel6 = "m3.medium"
```

Comment:
Why a different default instance type for RHEL? Seems like we should just make them the same.